### PR TITLE
Replace bisq with onbtc for about alliances

### PR DESCRIPTION
**Pull request by hunicus**
Screenshot from 2024-04-29 22-38-53

### CODE REVIEW

File: frontend/src/app/components/about/about.component.html [hunk 1]
```diff
@@ -343,8 +343,8 @@ <h3 i18n="about.alliances">Community Alliances</h3>
       <a href="https://opencrypto.org/" title="Coppa - Crypto Open Patent Alliance">
         <img class="copa" src="/resources/profile/copa.png" />
       </a>
-      <a href="https://bisq.network/" title="Bisq Network">
-        <img class="bisq" src="/resources/profile/bisq.svg" />
+      <a href="" title="Oficina Nacional del Bitcoin">
```
Comment (softsimon) [inline frontend/src/app/components/about/about.component.html:346]: If it's not going to be clickable (it's still clickable but go to the start page), better just remove the a link and add the title attribute to the img tag directly.

**Review by softsimon — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/about/about.component.html (hunk 1 above)
Comment (wiz) [reply frontend/src/app/components/about/about.component.html:346]: we can add the URL, just wait a few days

**Comment by hunicus** [comment]
Screenshot from 2024-05-12 17-20-05

**Review by softsimon — APPROVED** [review]
tested ACK @ [bf541f089]

**Pull request merged**